sigstore/_internal/oidc/ambient.py

```diff
@@ -23,10 +23,15 @@
 import requests
 from pydantic import BaseModel
 
-from sigstore._internal.oidc import IdentityError
+from sigstore._internal.oidc import _AUDIENCE, IdentityError
```

Comment:
Tiny nit: now that we're importing it somewhere else, maybe we should do `_AUDIENCE -> AUDIENCE`